### ISSUE TITLE
Add release notes / changelog for each version

### ISSUE THREAD

**Issue by Snailedlt** [jboss-javassist/javassist#414]
I'm missing a quick way to catch up on all the changes in a new release. Now I could just spend some time and look at the git changes, but having a shorter "release notes" text under each release, would make it much easier and faster to keep track of things, without all the clutter in a git log.

**Comment by chibash** [comment]
I added a short note to the release.  It is a copy of the text in Readme.html.

**Comment by chibash** [comment]
Yes, I will do so.  I also plan to change README.html into .md.  It is an html file because it was easy to read when it was first written.

**Comment by Snailedlt** [comment]
That would be great!
Thanks for the work you do :)
